schemas/Identity.json

```diff
@@ -0,0 +1,22 @@
+{
+    "type": "object",
+    "description": "The user properties to apply when identifying",
+    "properties": {
+      "eventName": {
+        "enum": ["Identity"]
+      },
+      "ftueUseCaseSelection": {
+        "description": "The selected messaging use case during the onboarding flow.",
+        "type": "string",
+        "enum": [
+          "FriendsAndFamily",
```

Comment:
@daniellekirkwood are there any preferred tracking values for the use case options?

property - `ftueUseCaseSelection`
options =
```
FriendsAndFamily
Teams,
Communities,
Skipped"
```